src/cmdline.c

```diff
@@ -96,7 +96,7 @@ int cmdline_run(const char **argv, int argc)
 	 * a null pointer in the array. Because of this, AppKit in OS X 10.10 will
 	 * dereference it, causing a segmentation fault.
 	 */
-	char** mutableArgv = malloc(argvsize);
+	const char** mutableArgv = malloc(argvsize);
```

Comment:
I reverted this because it was a warning in VS, how can it be a warning for GCC? It can't be const because you are freeing it.